cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/TopicReplicationFactorAnomaly.java

```diff
@@ -31,11 +31,14 @@
  */
 public class TopicReplicationFactorAnomaly extends TopicAnomaly {
   protected Short _targetReplicationFactor;
-  protected Set<String> _topicsWithBadReplicationFactor;
+  protected Map<Boolean, Set<String>> _badTopicsByFixability;
```

Comment:
Nit: The name `_badTopicsByFixability` is not clear -- can we rename it to represent what `bad` means?